packages/node_modules/@ciscospark/widget-meet/src/container.js

```diff
@@ -234,105 +258,122 @@ export class MeetWidget extends Component {
     return snapTransform[distances.indexOf(Math.min(...distances))];
   }
 
-  incomingCallNotification(incomingCall) {
+  incomingCallNotification(incomingCall, fromPersonName) {
     const {
       intl,
       avatarImage
     } = this.props;
     const {formatMessage} = intl;
     const details = {
-      username: incomingCall.from.person.name,
+      username: fromPersonName,
       message: formatMessage(messages.incomingCallMessage),
       avatar: avatarImage
     };
     this.props.createNotification(incomingCall.locusUrl, NOTIFICATION_TYPE_OTHER, details);
   }
 
-  render() {
+  render() { // eslint-disable-line complexity
```

Comment:
We might want to break these up into smaller functions at some point so it isn't as complex.